package.json

```diff
@@ -2,9 +2,10 @@
   "name": "citgm",
   "version": "7.2.2",
   "description": "The Canary in the Goldmine",
-  "homepage": "http://nodejs.org",
-  "main": "bin/citgm",
+  "homepage": "https://nodejs.org",
   "preferGlobal": true,
+  "type": "module",
+  "exports": {},
```

Comment:
Is this the right way to export nothing?

Reply:
LGTM. `false` should work also.